package.json

```diff
@@ -5,16 +5,9 @@
   "main": "main.js",
   "scripts": {
     "start": "electron .",
-    "dev": "electron . --enable-logging",
     "clean": "rm -rf ./dist",
-    "clean:osx": "rm -rf ./dist/osx",
```

Comment:
do you know how to build for mac using `electron-packager`? I read somewhere that it needs to be signed.